main.cpp

```diff
@@ -187,10 +187,10 @@ double Lorry::getSpeed()
 }
 void Lorry::calculateAndPrintRisk()
 {
-    for(int i=0;i<passengers;i++)
+    for(size_t i = 0; i < passengers; ++i)
```

Comment:
You should not use `passengers` as the thing that defines the number of elements in `passengerRiskArray[]
- [ ] use `passengerRiskArray.size();` as that will always report the correct size.